javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
The major concern with `instanceof` checks is that we can lose information here - does that matter (e.g. sub-classes of `Error` will lose additional properties and information like `TypeError`, etc).